src/op_handlers/ptr_add.rs

```diff
@@ -0,0 +1,19 @@
+use crate::{
+    ptr_operator::{ptr_operands_read, ptr_operands_store},
+    state::VMState,
+    value::FatPointer,
+    Opcode,
+};
+
+pub fn _ptr_add(vm: &mut VMState, opcode: &Opcode) {
+    let (pointer, diff, src0) = ptr_operands_read(vm, opcode, "ptr_add");
+    let (new_offset, overflow) = pointer.offset.overflowing_add(diff);
+    if overflow {
+        panic!("Offset overflow in ptr_add");
```

Comment:
Let's add a todo here for when we properly implement the vm's panic routine.

Reply:
Done